.env

```diff
@@ -0,0 +1 @@
+API_KEY="AIzaSyBldi9w74UpY7_4PU3-88fG1Hhw9iX13yk"
```

Comment:
Dont push .env file to public repository

Reply:
Yes, I'll make the changes. Thank you.